lib/AST/Type.cpp

```diff
@@ -3188,6 +3188,12 @@ TypeBase::getContextSubstitutions(const DeclContext *dc,
       continue;
     }
 
+    // Continue looking into the parent.
```

Comment:
Please don't add this -- instead ensure you're passing the right type into this function.